src/main/java/com/example/demo/model/LineItemId.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.model;
+
+import jakarta.persistence.Embeddable;
+
+import java.util.UUID;
+
+@Embeddable
+public record LineItemId(
+     UUID id
+) {
+
+    public static LineItemId generate() {
+        return new LineItemId(UUID.randomUUID());
```

Comment:
UUID에 집착해야 하는 아주 특별한 상황이 아니라면 그냥 이렇게 하면 됩니다.

```suggestion
        return new LineItemId(UUID.randomUUID().toString());
```